toy_reweight.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
```

Comment:
can you come up with an interesting example ? For example comparing the solution with a direct prox cd approach and a reweighted one in terms of timing, minimum found, prediction error, support recovery on a simulation ?